packages/regrade/regrade.0.1.0/opam

```diff
@@ -0,0 +1,40 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Automatic homework grader"
+description: "Simple grading system based on regular expressions."
+maintainer: ["Samuel Mimram <smimram@gmail.com>"]
+authors: ["Samuel Mimram <smimram@gmail.com>"]
+license: "GPL-3.0-only"
+homepage: "https://github.com/smimram/regrade"
+bug-reports: "https://github.com/smimram/regrade/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.13"}
+  "csv"
+  "re" {>= "1.1.0"}
```

Comment:
```suggestion
  "re" {>= "1.11.0"}
```

The release pages of re mentions the Pcre module in 1.11. Your code might be compatible with `1.10` or `1.9` or … `1.4`. You are welcome to bisect this version range to find a better bound, or we can try 1.11 which is possibly narrower than it needs to be but likely to be ok.